Source/Public/Add-VSTeamBuildDefinition.ps1

```diff
@@ -1,16 +1,27 @@
+# Creates a new definition.
+# Get-VSTeamOption 'build' 'Definitions'
+# id              : dbeaf647-6167-421a-bda9-c9327b25e2e6
+# area            : Build
+# resourceName    : Definitions
+# routeTemplate   : {project}/_apis/build/{resource}/{definitionId}
+# https://bit.ly/Add-VSTeamBuildDefinition

```

Comment:
What is this witchcraft? What did you do there or how? That is cool. Did you add every cmdlet here?

Reply:
As I reviewed each function I found the docs page and created the short link. I don't have one for each yet but I will continue to create as I review the functions.